pages/api/slackInvite.js

```diff
@@ -0,0 +1,29 @@
+export default function slackInvite(req, res) {
+  const email = req.body.email;
+
+  const body = {
+    channel_ids: process.env.ROCDEV_SLACK_CHANNEL_IDS,
+    email,
+    team_id: process.env.ROCDEV_SLACK_TEAM_ID,
+    custom_message:
+      "Come join the biggest development community in Rochester, NY",
+    resend: true,
+  };
+
+  return fetch("https://slack.com/api/admin.users.invite", {
+    method: "post",
+    headers: {
+      Authorization: `Bearer ${process.env.ROCDEV_SLACK_INVITE_API_TOKEN}`,
```

Comment:
Is this on the client side?

Reply:
Nope, server side.

Things that run on the server only (code is never shipped to the client) in Next.js
 * `getServerSideProps`
 * `getStaticProps` (build time)
 * `getStaticPaths` (build time)
 * Anything in the `pages/api` folder

Everything in `pages/` corresponds to a path in the app. The `pages/api` folder is special - instead of referring to a rendered page, each file in there corresponds to a node endpoint. So if you make a request to `localhost:3000/api/slackInvite`, that will be handled (on the server) by the default export from this file.